sorl/__init__.py

```diff
@@ -5,7 +5,6 @@
 
 __author__ = "Mikko Hellsing"
 __license__ = "BSD"
-__version__ = '12.6.0'
```

Comment:
Let's also use the `pkg_resources.get_distribution` method like in `docs/conf.py` in the `sorl/__init__.py` for the `__version__` variable?

Reply:
Do we want both `setup.py` and `sorl/__init__.py`? `__version__` is quasi standard anyway so both are, well, somewhat correct:

https://packaging.python.org/guides/single-sourcing-package-version/

Reply:
Yeah, we want both, please re-add the __version__ parameter as described here: https://github.com/pypa/setuptools_scm/#retrieving-package-version-at-runtime